display.js

```diff
@@ -368,3 +368,25 @@ export function printVulnerabilities(vulnerabilities) {
   }
   console.log(`${vulnerabilities.length} vulnerabilities found.`);
 }
+
+export function printSponsorBanner(options) {
+  if (
+    process?.env?.CI &&
```

Comment:
No idea why `process` is undefined in deno. Is this a regression with the latest deno?